deploy/class.yaml

```diff
@@ -0,0 +1,96 @@
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: freenas-nfs
+#  annotations:
+#    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: freenas.org/nfs
+allowVolumeExpansion: true
+parameters:
+  # namespace of the secret which contains FreeNAS server connection details
+  # default: kube-system
+  #serverSecretNamespace:
```

Comment:
I don't think you can share secrets across namespaces

Reply:
Let me test this out real fast and see what I find.  I'm not sure why a go client wouldn't be able to access secrets from any namespace..seems odd.

Reply:
I've confirmed that this does indeed work.  I changed the namespace on the storage class definition and watch provisioning fail several times, then created the secret in the desired namespace, the provisioner was able to retrieve and use the secret for provisioning.

Reply:
I think `ClusterRole`s have access to everything.

Reply:
👍